javascript-create-module/template/vite.config.mjs

```diff
@@ -5,21 +5,6 @@ import { spawnSync } from "node:child_process";
 export default defineConfig({
   plugins: [
     jahia({
-      client: {
```

Comment:
Shouldn't we keep the default settings (maybe commented) so developers starting a new project with `npm init @jahia/module@latest <module-name>` know how to tweak it?

Reply:
I'll do that, it's a great idea!

Reply:
Done in 2322ebc